src/components/PopupWithForm.js

```diff
@@ -33,14 +32,15 @@ export default class PopupWithForm extends Popup {
   setEventListeners() {
     this._popupElement.addEventListener("submit", () => {
       this._handleFormSubmit(this._getInputValues());
+      this._popupForm.reset();
 
       this.close();
     });
     super.setEventListeners();
   }
 
-  close() {
-    this._popupForm.reset();
-    super.close();
-  }
+  // close() {
+  //   this._popupForm.reset();
+  //   super.close();
```

Comment:
Please, uncomment this method


`this._popupForm.reset();` is needed according to the task in method `close` 


```js
  close() {
    this._popupForm.reset();
    super.close();
  }

``